src/runtime_src/tools/scripts/xrtdeps-win22.py

```diff
@@ -724,6 +734,63 @@ def getBuildAndInstallLibrary(self, verbose):
   def skipBuild(self):
     return self.skipBuildInstall == False
 
+#==============================================================================
+# -- Class: pybind11_package  -------------------------------------------------
+#==============================================================================
+class pybind11_package():
+    """
+    This class contains methods to check for a pybind11 installation and to
+    install the package using pip. Note that even if a pybind11 version exists,
+    if it doesn't match the expected_version parameter it will be overwritten.
+
+    Parameters
+    ----------
+    expected_version : str
+        Desired pybind11 version
+    """
+
+    def __init__(self, expected_version='2.10.4'):
+        self.installed = False
+        self.expected_version = expected_version
+        
+        if self.check_installed():
+            print(f"Successfully found pybind11 installation")
```

Comment:
Being that there is no format in the string, to prevent confusion with future developers, it is recommended not to us the 'f' option.
```suggestion
            print("Successfully found pybind11 installation")
```